README.md

```diff
@@ -49,24 +49,35 @@ and it will be used for waypoint being created:
 * `horizontal`
 * `triggerOnce`
 * `offset`
+* `contextElementId` (in place of context)
 * `continuous`
 
+An example of using a way point with a context (where you want to have the way point be based upon a "container" div offset instead of the body offset):
+
+```handlebars
+<div id="container" style="height:100px; overflow-y:scroll;">
+  <div style="height:500px;">
+    {{#waypoint contextElementId="container" offset="50" on-up="scrollingUp" on-down="scrollingDown"}}
```

Comment:
The acceptance tests confused me mucho since they were expecting action when no scrolling happened. (And they were failing/commented out). I just removed them, but if you could give me some guidance on what the expectation is I could fix)